src/app/security-group/sg-actions/sg-action.service.ts

```diff
@@ -1,20 +1,27 @@
-import { Injectable } from '@angular/core';
-import { ActionsService } from '../../shared/interfaces/action-service.interface';
 import { SecurityGroup } from '../sg.model';
-import { SecurityGroupAction } from './sg-action';
-import { SecurityGroupRemoveAction } from './sg-remove';
-import { SecurityGroupRulesAction } from './sg-rules';
 
+export enum SecurityGroupActionType {
+  View = 'view',
+  Delete = 'delete',
+}
+
+const SecurityGroupDeleteAction = {
```

Comment:
please add an interface for actions and add typing to service actions property.
Please, do this for account-actions.service.ts also,
Add some action model interface in a separate file